Editor.js

```diff
@@ -30,6 +31,16 @@ define([
 				self._focusedEditorCell = null;
 			});
 			this._listeners.push(this._editorFocusoutHandle);
+
+			aspect.before(this, 'refresh', function() {
```

Comment:
Originally I was concerned that some other mixin might destroy the dom before calling `inherited` in its own `refresh` method, but realistically there's probably no reason why it would.